public/components/table.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import FixedDataTable from 'fixed-data-table';
+import Axios from 'axios';
+
+const {Table, Column, Cell} = FixedDataTable;
+
+const TextCell = ({rowIndex, data, col}) => (
+    <Cell>
+        {data[rowIndex][col]}
+    </Cell>
+);
+
+class TableComponent extends React.Component
+{
+    constructor(props)
+    {
+        super(props);
+
+        this.state = {
+            rows: [],
+            pagination: this.props.pagination,
+            perPage: this.props.perPage,
+            hasNext: true,
+            headers: []
+        };
+    }
+
+    componentDidMount()
+    {
+        this.loadMore();
+    }
+
+    loadMore()
+    {
+        if (!this.state.hasNext) {
+            return;
+        }
+
+        Axios.get(`./person/${this.state.pagination}/${this.state.perPage}`).then(
+            res => {
+                const data = res.data;
+                this.setState((prevState, props) => {
+                    return {
+                        hasNext: Boolean(data['next_page_url']),
+                        rows: data.data,
+                        pagination: prevState.pagination + 1,
+                        perPage: prevState.perPage,
+                        headers: Object.keys(data.data[0])
+                    }
+                });
+            }
+        );
+    }
+
+    render()
+    {
+        const columns = this.state.headers.map((header, index) => {
+            return (
+                <Column
+                    key={index}
+                    header={<Cell>{header}</Cell>}
+                    flexGrow={1}
+                    width={100}
```

Comment:
Flexible width (how does that work with these data tables? Compute width when building?)